core/src/test/scala/unit/kafka/network/SocketServerTest.scala

```diff
@@ -1129,7 +1133,21 @@ class SocketServerTest {
       val expectedTotalTimeCount = totalTimeHistCount() + 1
       TestUtils.waitUntilTrue(() => totalTimeHistCount() == expectedTotalTimeCount,
         s"request metrics not updated, expected: $expectedTotalTimeCount, actual: ${totalTimeHistCount()}")
-
+      Thread.sleep(10)
```

Comment:
why sleep here?

Reply:
updated to use TestUtils.waitUntilTrue to have a more controlled wait time